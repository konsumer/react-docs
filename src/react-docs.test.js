/* global describe it expect */

import reactDocs from './index'

describe('react-docs', () => {
  it('should handle component in ES6', () => {
    expect(reactDocs('src/examples/component_es6.js')).toMatchSnapshot()
  })
  it('should handle createReactClass in ES5', () => {
    expect(reactDocs('src/examples/createReactClass_es5.js')).toMatchSnapshot()
  })
  it('should handle PureComponent in ES6', () => {
    expect(reactDocs('src/examples/purecomponent_es6.js')).toMatchSnapshot()
  })
  it('should handle stateless component in ES5 exported as default', () => {
    expect(reactDocs('src/examples/stateless_es5_default.js')).toMatchSnapshot()
  })
  it('should handle a stateless component in ES5 exported', () => {
    expect(reactDocs('src/examples/stateless_es5_export.js')).toMatchSnapshot()
  })
  it('should handle a stateless component in ES6', () => {
    expect(reactDocs('src/examples/stateless_es6.js')).toMatchSnapshot()
  })
})
