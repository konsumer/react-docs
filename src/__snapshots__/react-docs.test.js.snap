// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-docs should handle PureComponent in ES6 1`] = `undefined`;

exports[`react-docs should handle a stateless component in ES5 exported 1`] = `undefined`;

exports[`react-docs should handle a stateless component in ES6 1`] = `undefined`;

exports[`react-docs should handle component in ES6 1`] = `undefined`;

exports[`react-docs should handle createReactClass in ES5 1`] = `undefined`;

exports[`react-docs should handle stateless component in ES5 exported as default 1`] = `undefined`;
